src/contrib/cluster/Akka.Cluster.Tools.Tests/PublishSubscribe/DistributedPubSubMediatorSpec.cs

```diff
@@ -54,5 +56,58 @@ public void DistributedPubSubMediator_can_be_activated_in_child_actor()
                 a.ShouldNotBe(ActorRefs.Nobody);
             });
         }
+
+        [Fact]
+        public void DistributedPubSubMediator_should_get_dead_letter()
+        {
+            EventFilter.Exception<NullReferenceException>().Expect(0, () =>
+            {
+                var actor = Sys.ActorOf((dsl, context) =>
+                {
+                    IActorRef mediator = null;
+                    ICancelable cancelable = null;
+                    dsl.OnPreStart = actorContext =>
+                    {
+                        Sys.EventStream.Subscribe(context.Self, typeof(DeadLetter));
+                        mediator = DistributedPubSub.Get(actorContext.System).Mediator;
+                        mediator.Tell(new Subscribe("pub-sub", context.Self));
+                    };
+
+                    dsl.Receive<string>(s => s.Equals("UnSub"), (s, actorContext) =>
+                    {
+                        mediator.Tell(new Unsubscribe("pub-sub", context.Self));
+                    }); 
+                    dsl.Receive<SubscribeAck>((s, actorContext) =>
+                    {
+                        actorContext.System.Log.Info($"{s.Subscribe.Topic}");
+                        cancelable = context.System.Scheduler.ScheduleTellOnceCancelable(TimeSpan.FromSeconds(5), context.Self, "UnSub", context.Self);
```

Comment:
No need to use the scheduler here - use `ExpectMsg<T>` outside the actor for the test